repepo/paper/make_figures_rebuttals.py

```
@@ -0,0 +1,239 @@
+    ax.plot([0, 10], [0, 10], color="black", linestyle="--")
+    # Remove legend
+    ax.get_legend().remove()
+    fig.suptitle("Steerability vs RMSE for different datasets")
+    fig.savefig(f"figures/{model}_scatter_steerability_vs_rmse.pdf")
+
+
+make_plot_for_steerability_vs_mse(df, selected_datasets)
+
+# %%
+# Load the steering results for the selected datasets
+import pandas as pd
```

Ruff

Module level import not at top of file

(E402)